### DISCUSSION TITLE
[THEME] A simple adaptation of the Obsidian Theme --> Greensidian

### DISCUSSION THREAD

**Discussion by obuno** [TriliumNext/Trilium#4547] [Show and tell]
# Trilium Greensidian Theme
[Trilium Notes](https://github.com/zadam/trilium) Server custom theme.
- Dark theme based on [Trilium Obsidian Theme](https://github.com/greengeek/trilium-obsidian-theme/blob/main/README.md)

## Instructions
- Create a new note in Trilium notes.
- Under the Basic Properties tab, set Note type to Code and choose the CSS option
- Under the next tab over, called Owned attributes, paste in the following: #appTheme=Greensidian
- In the body of the notes paste the contents of greensidian.css
- Click on the Trilium icon at the top left of the page and choose Options
- Select Appearance and choose Greensidian in the drop down under Theme

If you do not see Greensidian as an option, try refreshing the page in your browser. 

## Get it here
https://github.com/obuno/trilium-greensidian-theme

## Preview
![greensidian3](https://github.com/zadam/trilium/assets/32042396/114c1bcd-c373-4149-acd4-4d24fcf06b44)
![greensidian2](https://github.com/zadam/trilium/assets/32042396/f42a9a76-2f6f-4527-b0b4-ad0c93a9cbd9)
![greensidian1](https://github.com/zadam/trilium/assets/32042396/24de7f6e-93c4-4efb-888e-c03334534b66)

Thanks for Trilium, it rocks !

**Comment by rodrigoalexandrecosta** [comment]
Love it! I like green very much, I found it very nice on my eyes. Thank you!

Edit: this could certainly be listed in [awesome trilium](https://github.com/Nriver/awesome-trilium)

**Reply by Nriver** [reply]
Sure
